packages/nav-frontend-tabell-style/md/Tabell.overview.mdx

```diff
@@ -21,19 +21,19 @@ Legg til stilarket fra pakken og sett CSS-klassen `tabell` på HTML-tabellen din
     </thead>
     <tbody>
       <tr>
-        <td>1</td>
+        <th>1</th>
```

Comment:
Skulle vi droppe dette? Ref: https://nav-it.slack.com/archives/GTSNJ1WF2/p1606305279001600?thread_ts=1605709905.006700&cid=GTSNJ1WF2

Reply:
Ja dropper det. Legger heller til et eksempel i dokumentasjonen